README.md

```diff
@@ -24,18 +24,18 @@ This is a simple configuration reading tool. It just does the following:
 
 - [Installation](#installation)
 - [Usage](#usage)
-    - [Read Configuration](#read-configuration)
-    - [Read Environment Variables Only](#read-environment-variables-only)
-    - [Update Environment Variables](#update-environment-variables)
-    - [Description](#description)
+  - [Read Configuration](#read-configuration)
```

Comment:
Please roll back the formatting. Different IDEs do autoformatting differently, and I see no point in changing it back and forth.